### DISCUSSION TITLE
How to cancel workflow instance?

### DISCUSSION THREAD

**Discussion by rairafaqatali** [elsa-workflows/elsa-core#1497] [Q&A]
Hi @sfmskywalker 

I have created a workflow instance that has some blocking activities in it.

I want to send that workflow instance to cancel state.

Can you let me know how can I cancel that workflow instance by using the visual designer?

**Comment by sfmskywalker** [comment]
There's currently no way to cancel a workflow from the dashboard, but as per [this Q&A](https://github.com/elsa-workflows/elsa-core/discussions/1480) we will have it once #1495 is done.
